### PR TITLE
676 show all dao holdings

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "DAI": "0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359",
+  "GUSD": "0x056fd409e1d7a124bd7017459dfea2f387b6d5cd",
+  "KNC": "0xdd974D5C2e2928deA5F71b9825b8b646686BD200",
+  "PAX": "0x8e870d67f660d95d5be530380d0ec0bd388289e1",
+  "TUSD": "0x0000000000085d4780B73119b644AE5ecd22b376",
+  "USDC": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  "USDT": "0xdac17f958d2ee523a2206206994597c13d831ec7"
```
[the remark this answers is not in the document]
Comment (tibetsprague) [reply data/tokens.json]: isn't this good enough for now? why do we need the decimals to look up the balances?

[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply data/tokens.json]: we need the decimals to SHOW the balances. I.e. if PX uses 16 isntead of 18 decimals, the `fromWei` functino will show `0,01 PAX` when it shoudl show `1 PAX`.

[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply data/tokens.json]: if you're sure these tokens all use 18 decimals, we can leave it like this (but add some comment on the top of the list to warn devs that want to add new tokens)

**Comment by jellegerbrandy** [comment]
i cannot approve this pr since I created it, but you @tibetsprague  can :-)

File: data/tokens.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jellegerbrandy) [reply data/tokens.json]: forget about adding the comment, I created this issue: https://github.com/daostack/alchemy/issues/753

**Comment by tibetsprague** [comment]
ah you are right, they are not all 18 decimals. ok ill do this

**Comment by jellegerbrandy** [comment]
@tibetsprague what is the status of this PR? Afaics it still needs:

- a resolution of a bug in the client for getting the balances (did you open an issue for that specifically?)
- take into account decimals in visualizing the balanaces

**Comment by tibetsprague** [comment]
yes exactly. I didn't create a ticket because I wanted to test with the latest client, but I will

**Comment by tibetsprague** [comment]
@jellegerbrandy ready to go?

**Comment by jellegerbrandy** [comment]
I cannot leave a review as I created the PR, but for me it is good to go.

**Pull request merged**